src/client/containers/DeleteBoardModal/DeleteBoardModal.container.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './DeleteBoardModal.style.css';
+import CloseButton from '../../components/CloseButton/CloseButton.component';
+import PageTitle from '../../components/PageTitle/PageTitle.component';
+import GenericButton from '../../components/GenericButton/GenericButton.component';
+
+const DeleteBoardModal = ({ boardInfo, setModalVisibility }) => {
+  // const { userId } = useParams();
+
+  const userId = 2;
```

Comment:
the user comes from useUser()

Reply:
And also since the params and the useUser() will also come from  the respective boards that it will be nested in, should I just delete them , and . I did not realise it before.

Reply:
yes 👍